.github/actions/install-python-packages/action.yml

```diff
@@ -7,6 +7,7 @@ runs:
       run: |
         sudo pip3 install -U \
           black==22.3.0 \
+          empy==3.3.4 \
```

Comment:
Instead of doing this, you can use eProsima-CI to install colcon. See for instance:
* eProsima/Fast-DDS#4075